Devices/mfdevice.schema.json

```diff
@@ -127,7 +127,7 @@
             "$id": "#root/Pins/items/Id",
             "title": "Id",
             "description": "The identifier for the message type. Unique in the scope of the device.",
-            "type": "integer"
+            "type": "string"
```

Comment:
The code expected a string, not an integer. Updated the schema to match.

Reply:
I tried changing this to an integer in the class and it's crazy how much stuff breaks. It gets used all over the place as the value in dropdowns, saved to config files as strings, gets passed to Display(), etc.

It's not possible to change it to an integer without major changes throughout the code, so leaving it as a string.

Reply:
I think you even intended it to be a string, there's unit tests that explicitly used a string for it:

```C#
            o1.CustomName = "DisplayPin";
            o1.CustomType = "CustomType";
            o1.Value = "Value";
            o1.MessageType = "MessageType";
            Assert.IsFalse(o1.Equals(o2));
 ```

Reply:
These are how many files get touched trying to change it to an int instead of a string:

<img width="311" alt="image" src="https://github.com/MobiFlight/MobiFlight-Connector/assets/9524118/f9a43414-7a99-40c9-9562-d85c606f9bd7">

Reply:
Hmhm, the FW reads the messageID as integer: `int16_t messageID = cmdMessenger.readInt16Arg();`

Reply:
Oh, just checked again. It always send as a string, `readInt16Arg()` converts it within the cmdMessenger.